osgpkitools/incommon_request.py

```diff
@@ -174,14 +171,18 @@ def test_incommon_connection(config, restclient):
     logger.debug('response text: ' + str(response_text))
     try:
         if response.status == 200:
-            print("Connection successful to InCommon API") 
+            print(prog + ": " + "Connection successful to InCommon API") 
         else:
             # InCommon API HTTP Error codes and messages are not consistent with documentation.
-            # {Unknown user}
-            print("Connection failure to InCommon API. Check your authentication credentials.")
+            print(prog + ": " + "Connection failure to InCommon API")
+
+            if response.status == 401:
+                print("Check your authentication credentials")
+
+        print("HTTP " + str(response.status) + " " + str(response.reason))
```

Comment:
This should be indented further to only show on failures.

Reply:
I thought you meant the HTTP code and message for all cases.

Reply:
We don't need to see the 200 OK message.  It doesn't matter a whole lot since this is just for -t, but it's better that way.